airbyte-integrations/connectors/destination-snowflake/README.md

```diff
@@ -22,7 +22,7 @@
 Put the contents of the `Snowflake Integration Test Config` secret on Rippling under the `Engineering` folder into `secrets/config.json` to be able to run integration tests locally.
 
 1. Put the contents of the `destination snowflake - insert test creds` LastPass secret into `secrets/insert_config.json`.
-1. Put the contents of the `destination snowflake - insert staging test creds` secret into `insert_staging_config.json`.
+1. Put the contents of the `destination snowflake - insert staging test creds` secret into `internal_staging_config.json`.
```

Comment:
Should this be `secrets/internal_staging_config`?

Reply:
Have no idea :) I just aligned documentation to what we actually have a codebase and secret manager